src/api/routes.js

```diff
@@ -47,12 +47,13 @@ export default function configureRoutes( key, rootPath, options = {} ) {
     getRouteConfig: ( methodKey, ...args ) => {
       const config = routes[ methodKey ]( ...args );
       return typeof config === 'string'
-        ? { method: methodDict[ methodKey ], route: config, getData: options.getData || ( data => data ) }
+        ? { method: methodDict[ methodKey ], route: config, getData: options.getData || ( data => data ), onError: options.onError || (error => error) }
```

Comment:
@maxwell-oroark i'm thinking our no-op should return a rejected promise so that any `catch` handlers down the line will be invoked